.cirrus.yml

```diff
@@ -302,6 +302,23 @@ task:
       script:
         - ./dev/bots/firebase_testlab.sh
 
+    - name: gold_tryjob_status # linux- pre-submit- only
```

Comment:
nit: I think the convention is to postfix this with `-linux` in the job name so it's obvious from the PR what OS it's running on.